frame/merkle-mountain-range/src/tests.rs

```diff
@@ -259,8 +262,9 @@ fn should_generate_proofs_correctly() {
 			)
 		);
 		assert_eq!(
-			proofs[4],
+			proofs[5],
```

Comment:
this change will have to be reverted, or the leaves changed, as done for the assertion for `proofs[6]`, since these adjustments were consequent of the duplicate `leaf_index = 0` entry necessitated by still starting the proofs range from 0.